pkg/registry/apis/provisioning/repository/local_test.go

```diff
@@ -12,16 +14,62 @@ import (
 func TestLocalResolver(t *testing.T) {
 	resolver := &LocalFolderResolver{
 		PermittedPrefixes: []string{
-			"/github/testdata",
+			"github",
 		},
-		HomePath: "/",
+		HomePath: "./",
 	}
 
-	_, err := resolver.LocalPath("github/testdata")
+	fullpath, err := resolver.LocalPath("github/testdata")
 	require.NoError(t, err)
+	require.Equal(t, "github/testdata", fullpath)
 
 	_, err = resolver.LocalPath("something")
 	require.Error(t, err)
+
```

Comment:
These tests are not really required... but when things stopped working I thought it was based on how we read things out of local storage.

The real cause was that the folder manager created folders without a trailing "/" so they become files and then fail to be read as directories